modules/mobile-guides/pages/intro.adoc

```diff
@@ -1,6 +1,6 @@
 = Build Mobile Apps
 :page-role: tiles -toc
-:description: Use App Services in combination with Couchbase Mobile to develop mobile apps.
+:description: Use App Services in combination with Couchbase Lite to develop mobile apps.
```

Comment:
Suggestion: Use App Services in Capella in combination with Couchbase Lite embedded NoSQL database to develop mobile apps.